### PR TITLE
Update baggers.js

### PR DESCRIPTION
**Pull request by fredericpot**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -13299,6 +13299,42 @@ var data = {
...
+          "lang": [
+            "fr"
+          ]
+        }
+      ],
+      "cities": [
+        "Nantes"
+      ],
+      "contacts": {
+        "twitter": "@fpot",
```
Comment (dadoonet) [inline baggers.js]: Can you remove the @ sign please?

File: baggers.js [hunk 2]
```diff
@@ -13299,6 +13299,42 @@ var data = {
...
       }
+    },
+    {
+      "since": "2016-07-18",
+      "name": "Frédéric Pot",
+      "bio": "Co-fondateur de Lucca en 2002, je suis le Product Owner de la solution Figgo utilisée par 115 000 salariés de 900 sociétés à travers 30 pays. Je serai ravi de vous partager mon expérience sur les 15 dernières années autour de l’évolution dans la gestion d’un logiciel RH.",
+      "picture": "https://avatars1.githubusercontent.com/u/2751422?v=3&u=93aa7a0037a926090eb975aaf590840a069f6a44&s=140",
+      "websites": [
+        {
+          "name": "Lucca",
```
Comment (dadoonet) [inline baggers.js]: Name should conform with doc. Read https://github.com/brownbaglunch/bblfr_data/blob/gh-pages/README.md

**Comment by dadoonet** [comment]
I left some comments. Could you update your PR?

**Comment by fredericpot** [comment]
I updated my PR.

Thanks

**Pull request merged**